internal/k8s/registry.go

```diff
@@ -17,6 +17,9 @@ import (
 const annotationRegistry = "tilt.dev/registry"
```

Comment:
might comment this too, and put a little more detail in both comments: "Key for retrieving registry from node annotations if cluster created via Tilt's script <link> || Kind's script <link>"

Reply:
also maybe call this `tiltAnnotationRegistry`?

Reply:
done